src/scripts.js

```diff
@@ -87,41 +122,42 @@ function getBookingData(customer) {
     const dateMonth = date.getUTCMonth() + 1
     const dateYear = date.getUTCFullYear()
     const currentDate = dateYear + "-" + dateMonth + "-" + dateDay
-    let pic
     sidebar.innerHTML = `<h4>Total Spent: $${customer.totalSpent}</h4>`
+    let pic
     customer.bookings.forEach( booking => {
-    if (!booking.pic) {
-    pic = roomImages[Math.floor(Math.random() * roomImages.length)]
-    booking.pic = pic.toString()
-    }
- 
```

Comment:
this wasnt working, although I could probably go back and make booking pics stay consistent after getting a random one